module6/lessons/Week1/JobHuntHustle.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Job Hunt Hustle Intro
+---
+
+## Introduction
+
+Find job you're interested in, apply to job, interview for job, receive offer: that's the process right? Sometimes that works, but more often than not the path towards a job offer is more winding and involves more "hustle" on your end.
+
+What do I mean by "hustle"? I think of the hustle as putting in extra time and effort to help you stand out in comparison to the other candidates.
+
+There are so many different activities this can include, and by the end of today we will have a list of "Job Hunt Hustle" activities to choose from. 
+
+For "Job Hunt Hustle" we are not going to focus on things that help you ace the interview or grow your qualifications. We instead want to focus on things that help you move closer on the path towards jobs you want. Things that that help get your foot in the door and help you stay top of mind to potential employers.
+
+### Examples of Job Hunt Hustle
+
+#### Example 1
+[Thoughts from Ian Douglass, a former Turing Instructor](https://twitter.com/iandouglas736/status/1593731985142226944)
+
+#### Example 2
+[Thoughts from a LinkedIn Career Coach](https://www.linkedin.com/feed/update/urn:li:activity:7121159930262261760/)
+
+#### Example 3
+A snippet of the path towards Zoe's first development role:
+
+![AppNexus 0](/assets/images/module6/week1/AppNexus0.png)
+![AppNexus 1](/assets/images/module6/week1/AppNexus1.png)
+![AppNexus 2](/assets/images/module6/week1/AppNexus2.png)
+
+#### Example 4
+A snippet of the path towards Zoe volunteering with Code.org for a week while in college (definitely a bit cringy to look back on my old messages, but I think it's a solid example of thoughtful and genuine persistance paying off):
+
+For additional context Hadi Partovi is the CEO of Code.org, a non-profit that creates coding tutorials for K-12 students and pushes to expand k-12 computer science education. They organize The Hour of Code each December.
+
+![Code.org 1](/assets/images/module6/week1/Code1.png)
+![Code.org 2](/assets/images/module6/week1/Code2.png)
+![Code.org 3](/assets/images/module6/week1/Code3.png)
+![Code.org 4](/assets/images/module6/week1/Code4.png)
+
+<section class="call-to-action" markdown="1">
+Take a look through the above examples as starting points, then with your partner brainstorm ideas for how to "Job Hunt Hustle". Are there any "Job Hunt Hustle" strategies that have worked for you in the past? When we come back together your instructor will call on groups to share out ideas and compile a mega-list of ideas in Slack.
+</section>
+
+### Conclusion
+

```

Comment:
I also feel like there is something here that should be said about how identity impacts the way hustle is often perceived, but I don't think I have space right now to thoughtfully think about how to approach that. I feel like it might be related to identity in the job search. But I think it is probably best to not go there right now unless you think it's important to include something.

Reply:
Yea, I agree that it is an important topic, but I don't know that we have the expertise to dive into it in a lesson-plan kind of way... but we could open some space for students to just talk about how they are feeling about starting their job-hunt hustle.  I think those discussions could lead to some more organic insights.

Reply:
I like the idea of opening up a conversation at the end about how students are feeling about the job hunt hustle. Maybe in a think pair share style. I'll add that in.